src/pages/MyGardens/index.tsx

```diff
@@ -24,54 +26,61 @@ const useStyles = makeStyles({
 export const MyGardens = () => {
   const classes = useStyles();
   const { userData } = useUserState();
-  const [myGardens, setMyGardens] = useState(Array<Garden>());
+  const [isFetchingGardens, setIsFetchingGardens] = useState(true);
+  const [myGardens, setMyGardens] = useState<Array<Garden>>([]);
 
   useEffect(() => {
     const getDataFromBackend = async () => {
-      if (userData?.id) {
+      if (userData.isLoggedIn && userData.id) {
         // TODO: FIX DATABASE CALL AFTER MVP
         const res = await axios.get(
           `https://the-fibonacci-api-staging.herokuapp.com/api/v1/gardens/userid/${userData?.id}`
         );
         setMyGardens(res.data || []);
+        setIsFetchingGardens(false);
       }
     };
 
     getDataFromBackend();
   }, []);
 
   return (
-    <div className="my-gardens-container">
-      <h1>My Gardens</h1>
-      <div className="gardens-view">
-        {myGardens.map((garden, index) => {
-          return (
-            <Link to={`/user/gardenView/${garden._id}`} key={index}>
-              <Card className={`garden-card ${classes.root}`}>
-                <CardActionArea>
-                  <CardMedia
-                    className={classes.media}
-                    image={gardenImage}
-                    title="Contemplative Reptile"
-                  />
-                  <CardContent>
-                    <Typography gutterBottom variant="h5" component="h2">
-                      {garden.name}
-                    </Typography>
-                    <Typography
-                      variant="body2"
-                      color="textSecondary"
-                      component="p"
-                    >
-                      {garden.description}
-                    </Typography>
-                  </CardContent>
-                </CardActionArea>
-              </Card>
-            </Link>
-          );
-        })}
+    <>
+      <Header />
+      <div className="my-gardens-container">
+        <LoadingWrapper isLoading={isFetchingGardens}>
+          <h1>My Gardens</h1>
```

Comment:
Loading wrapper for my gardens content
![MyGardens-loading](https://user-images.githubusercontent.com/22023256/131701887-8e7bb378-2d73-4e5b-ae26-a64d6d249a29.gif)